src/app.module.ts

```diff
@@ -1,9 +1,26 @@
 import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { ConfigModule } from '@nestjs/config';
```

Comment:
I feel that inserting this line right after line number 2 is more appealing :p 
so that libraries and packages are placed first, followed by the subsequent produced components and modules.

Reply:
it's just a code style to follow but not necessary if the team agrees to follow this practice we must add a task in ClickUp to add this rule in eslint configuration so please @souibguimohamed  add it as a task in clickup and we will discuss in next daily. It hasn't any relation with this RP

Reply:
Alright, OK.